build.sbt

```diff
@@ -428,8 +428,9 @@ lazy val metals = project
       // for fetching ch.epfl.scala:bloop-frontend and other library dependencies
       "io.get-coursier" % "interface" % V.coursierInterfaces,
       // for logging
-      "com.outr" %% "scribe" % "3.1.9",
-      "com.outr" %% "scribe-slf4j" % "3.1.9", // needed for flyway database migrations
+      "com.outr" %% "scribe" % "3.3.1",
+      "com.outr" %% "scribe-file" % "3.3.1",
```

Comment:
The file related stuff that we use got extracted out into it's own module and published separately. So that's the reason for this addition.